### PR TITLE
Task 7 simple template form

### PR DESCRIPTION
**Pull request by rajinder-yadav**

### CODE REVIEW

File: form-new-user/src/app/app.component.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+@Component({
+  selector: 'app-root',
+  template: `
+  <div class="pa2">
+    <div class="ba br4 p2 pl3 mb3 bg-purple white">
+      <h1>{{title}}</h1>
+    </div>
+    <div class="center">
+    <app-form-new-user></app-form-new-user>
```
Comment (stevenkampen) [inline form-new-user/src/app/app.component.ts]: @rajinder-yadav Just some indentation missing here.

**Comment by stevenkampen** [comment]
@rajinder-yadav LGTM, except for one small indentation issue.

**Comment by stevenkampen** [comment]
@rajinder-yadav Actually one more thing in the UI. The form only takes half the width of the screen, whereas the table takes 100%. When the width is constricted, the form labels get cut off much earlier than they need to be.

I guess we should always make sure the layout looks ok on smaller width screens, since these are designed to be used with the console open.

File: form-new-user/src/app/app.component.ts (hunk 1 above)
Comment (rajinder-yadav) [inline form-new-user/src/app/app.component.ts]: Good catch!

**Comment by rajinder-yadav** [comment]
Ya I purposely set the form to take 50% of the width, we can play around with this I agree. I'll talk to Fatima about this and general UI look & feel.

Thanks for the great input, I had my head down and was just rolling away!

**Comment by rajinder-yadav** [comment]
Changed form width to take up 100% for now.

**Pull request merged**